PR for milestone two

leonmezu1 opened the pull request:
## The merging requirements for this milestone are:

``` 
1. Prepare the directory structure
    - Create a `components` directory in the `src` folder.
    - Create four files in there, each one representing one React component. 
2. Implement the **App** component
    - **App** should implement the `render` function.
    - The `render` function should display **Display** and **ButtonPanel**.
3. Implement the **Display** component
    - It should accept the result of the calculation as an incoming prop.
    - The result prop should be a String.
    - The default value of the result prop should be 0.
4. Implement the **Button** component
    - It should accept a button name as a prop.
    - The button name prop should be a String.
5. Implement the **ButtonPanel** component
    - It should render the calculator panel with all the buttons in the following groups:
        - Group 1: AC, +/-, %, ÷
        - Group 2: 7, 8, 9, X
        - Group 3: 4, 5, 6, -
        - Group 4: 1, 2, 3, +
        - Group 5: 0, ., =
```

The Heroku deploy can be found here ➡️ https://leonmezu-react-calc.herokuapp.com/

leonmezu1 committed:
The required components were added

Lint results are OK and the components structure is the required

leonmezu1 committed:
Stickler config added

bellom commented on src/components/ButtonPannel/ButtonPanel.js:
```diff
@@ -0,0 +1,39 @@
+      <Button content="." />
+      <Button content="=" />
+    </div>
+  </div>
```
### [Optional]
I think we could make use of two loops to make the Button instead of repeating ourselves. let's practice [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) code.

To do this; create an array of symbols and loop through the array using [map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) to create the ButtonPanel component.

bellom commented on .stickler.yml:
```diff
@@ -0,0 +1,3 @@
+linters:
```
Kindly remove the `stickler` file
 
**NOTE:** Microverse has moved on to Github actions for continuous deployment please use that instead of stickler. 
![image](https://user-images.githubusercontent.com/31897434/85197008-14353c00-b2d6-11ea-932b-d3a0ab291c2a.png)

https://github.com/microverseinc/linters-config/tree/master/react-redux

bellom reviewed (changes requested):
## Hi @leonmezu1  :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 
 
**1. Let's make your repo looks professional by applying all the 3 basic rules, kindly update your `README` file and remove unnecessary content**.
Checkout out the below link for help, free free to follow this [template](https://github.com/microverseinc/readme-template) from pathwright

**Note: Make sure you commit often, you're having just 2 commits in on PR** 
example; 
![image](https://user-images.githubusercontent.com/31897434/94486417-4e688080-01d7-11eb-9773-7f9357901c6f.png)


![image](https://user-images.githubusercontent.com/31897434/92685488-b4c45680-f32f-11ea-9bea-222a21c096a6.png)

https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/

**2. Kindly check out my comments below;**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

leonmezu1 committed:
Suggested fixes by TSE are now implemented

stickler.yml has been deleted, the readme file is now updated and the ButtonPanel component is now a class component with loop for inner components

Oghenebrume50 reviewed (approved):
### APPROVED

Hi @leonmezu1 this looks good, kudos for making changes as directed by the previous reviewer please merge and move forward.
**This is approved** :+1: 

**RAPHAEL NORIODE** :mask: STAY_SAFE
